packages/examples/react-webpack-vite/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-webpack-vite",
+  "private": true,
+  "version": "1.0.0",
+  "workspaces": [
+    "host",
+    "remote"
+  ],
+  "license": "MulanPSL-2.0",
+  "scripts": {
+    "build": "pnpm --parallel --filter \"./**\" build",
+    "serve": "pnpm --parallel --filter \"./**\"  serve",
+    "build:remotes": "pnpm --parallel --filter \"./remote\" build",
+    "serve:remotes": "pnpm --parallel --filter \"./remote\" serve",
+    "dev:hosts": "pnpm --filter \"./host\" dev",
+    "stop": "kill-port --port 5000,5001"
+  },
+  "devDependencies": {
+    "@originjs/vite-plugin-federation": "^1.2.3",
```

Comment:
Using a version like ^1.2.3 is not so appropriate, because we may upgrade to version 2.x on that day, which may make it impossible to update, and more should use a version of workspace

Reply:
you are right. I was working on it locally and forget to reverted back god catch.